tests/ops/util/test_tcf_privacy_declarations.py

```diff
@@ -140,13 +161,12 @@ def test_privacy_declaration_purpose_overrides(
             if declaration.purpose
         }
         assert original_legal_basis == {
-            9: "Legitimate interests",
```

Comment:
i didn't totally understand why this isn't included in the `original_legal_basis` even after looking closer at the `get_tcf_base_query_and_filters` code, but i was following the example for purpose 2, and this does seem to be the behavior.

i assume it's expected! would be nice for me to understand _why_ though. based on my static analysis of the code, i thought that we'd still get the original legal basis populated in the underlying query, but i must be missing something :)

Reply:
ah, the main data I get from `get_tcf_base_query_and_filters` are the declarations that should show up in the TCF experience.  `original_legal_basis` is showing the original legal basis of declarations that met the criteria to be included in the experience.

Perhaps renaming the test variable would increase clarity around what I was testing here? To see original declarations, you could assert the legal bases on the declarations in the emerse system fixture

Reply:
I can just make a note in my TODO list for moving this work into the override to add clarity to this test, not worth pushing another commit here

Reply:
sure that sounds good! i think i get it :)